.github/workflows/main.yml

```diff
@@ -46,8 +46,8 @@ jobs:
 
       - name: Prep kolibri dist
         env:
-          KOLIBRI_WHL_URL: 'https://github.com/learningequality/kolibri/releases/download/v0.16.0-beta1/kolibri-0.16.0b1-py2.py3-none-any.whl'
-          KOLIBRI_WHL_FILENAME: "kolibri-0.16.0b1-py2.py3-none-any.whl"
+          KOLIBRI_WHL_URL: 'https://github.com/learningequality/kolibri/releases/download/v0.16.0-beta6/kolibri-0.16.0b6-py2.py3-none-any.whl'
+          KOLIBRI_WHL_FILENAME: "kolibri-0.16.0b6-py2.py3-none-any.whl"
```

Comment:
Excellent, hopefully this doesn't regress anything in kolibri-explore-plugin. Since we don't have tests, each time we bump Kolibri in our wrapper apps we have to do careful testing.